apps/web/src/components/navigation-menu/index.tsx

```diff
@@ -151,26 +165,55 @@ function NavigationMenu(props: NavigationMenuProps) {
     "sidebarHiddenItems:colors",
     db.settings.getSideBarHiddenItems("colors")
   );
+  const [currentTab, setCurrentTab] = useState<(typeof tabs)[number]["id"]>(
+    tabs.find((tab) => location.includes(tab.id))?.id || "home"
+  );
+  const notes = useStore((store) => store.notes);
+
+  useEffect(() => {
+    const setCounts = async () => {
+      const totalNotes = await db.notes.all.count();
+      const totalFavorites = await db.notes.favorites.count();
+      const totalReminders = await db.reminders.all.count();
+      const totalTrash = (await db.trash.all()).length;
+      const totalMonographs = await db.monographs.all.count();
+
+      console.log("totalNotes", totalNotes);
+      console.log("totalFavorites", totalFavorites);
+      console.log("totalReminders", totalReminders);
+      setRoutes((routes) => {
+        return routes.map((route) => {
+          switch (route.id) {
+            case "notes":
+              return { ...route, count: totalNotes };
+            case "favorites":
+              return { ...route, count: totalFavorites };
+            case "reminders":
+              return { ...route, count: totalReminders };
+            case "trash":
+              return { ...route, count: totalTrash };
+            case "monographs":
+              return { ...route, count: totalMonographs };
+            default:
+              return route;
+          }
+        });
+      });
+    };
+
+    setCounts();
+  }, [notes]);
```

Comment:
Don't think this is the best way to get and update counts for the home tab

Reply:
This should be okay for now.